extensions/hints/src/use-flow-page.tsx

```diff
@@ -0,0 +1,171 @@
+    if (hasRequiredError(searchText)) return;
+
+    console.log(searchText);
+
+    const toast = await showToast({ title: "Sending to Hints...", style: "ANIMATED" as any });
+
+    fetch(
+      `https://europe-west3-slipbox-6f705.cloudfunctions.net/createHintsFromRaycast?api_key=${preferences.api_token}`,
+      {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({ content: searchText }),
+      }
+    );
+    toast.style = Toast.Style.Success;
+    toast.title = "Successfully sent";
+  }
+
+  // ToDo add custom filtering to filter list of flows
+  // filterList(items.filter((item) => item.includes(searchText)));
+
+  function handleSubmitWithTag(flowHashtag: string) {
+    const messgaeWithHashtag = `${searchText} #${flowHashtag}`;
+    console.log(messgaeWithHashtag);
+    handleSubmit(messgaeWithHashtag);
+    setSearchText(``);
+  }
+
+  async function handleSearchFromClipboard() {
+    let selectedText;
+    try {
+      selectedText = await getSelectedText();
+    } catch (err) {
+      console.log(err);
+    }
+    if (selectedText) {
+      setSearchText(selectedText);
+    } else {
+      console.log("no selected text");
+    }
+  }
+
+  return (
+    <List
+      filtering={false}
+      isLoading={isLoading}
+      searchText={searchText}
+      onSearchTextChange={setSearchText}
+      navigationTitle="Send commands to Hints AI Assistant"
+      searchBarPlaceholder="Type your text here"
+      throttle={true}
+    >
+      <List.Item
+        title=""
+        subtitle={`Press Enter to send command to AI Assistant ${hashtag ? `in ${hashtag}` : ""}`}
+        actions={
+          <ActionPanel title="Actions">
+            <Action title="Send Command" onAction={() => handleSubmit(searchText)} />
+            <Action
+              title="Paste from Clipboard"
+              onAction={handleSearchFromClipboard}
+              shortcut={{ modifiers: ["cmd", "shift"], key: "v" }}
+            />
+            <Action.OpenInBrowser
+              title="Open Examples in F.A.Q."
+              url="https://hintsapp.notion.site/Hints-Flow-FAQ-4c62567028684fbbbe3e6be1afb738e9"
+              shortcut={{ modifiers: ["cmd"], key: "h" }}
+            />
+          </ActionPanel>
+        }
+      />
+
+      {pages ? (
+        <List.Section title="(Optional) Choose integration to send command">
+          {pages.map((flow: Flow) => (
+            <List.Item
+              key={`search-result-page-${flow.id}`}
+              title={`#${flow.tagName}`}
+              icon={{ source: logoIcons[flow.destination] }}
+              subtitle={`→ ${flow.destination}`}
+              keywords={[flow.destination]}
+              accessories={[{ text: "active" }]}
+              actions={
+                <ActionPanel title="Actions">
+                  <Action title="Send Command" onAction={() => handleSubmitWithTag(flow.tagName)} />
```

Comment:
[Missing an icon](https://developers.raycast.com/api-reference/user-interface/icons-and-images)